Pract.01/Task8.cpp

```diff
@@ -0,0 +1,26 @@
+// Task8.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+using namespace std;
+#include <iostream>
+
+int main()
+{
+    double pi = 3.14159265;
+    int r;
+    cout << "r: ";
+    cin >> r;
+
+    cout << "d: " << r * 2; //I assume that length of a circle means diameter 
```

Comment:
Check the formula in google and fix it. https://www.calculat.org/bg/%D0%BB%D0%B8%D1%86%D0%B5-%D0%BE%D0%B1%D0%B8%D0%BA%D0%BE%D0%BB%D0%BA%D0%B0/%D0%BE%D0%BA%D1%80%D1%8A%D0%B6%D0%BD%D0%BE%D1%81%D1%82/